.github/actions/isPullRequestMergeable/isPullRequestMergeable.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('underscore');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+const {promiseWhile} = require('../../libs/promiseWhile');
+
+const run = function () {
+    const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+    const pullRequestNumber = Number(core.getInput('PULL_REQUEST_NUMBER', {required: true}));
+
+    let isMergeable = false;
+    let mergeabilityResolved = false;
+    console.log(`Checking the mergeability of PR #${pullRequestNumber}`);
+    return promiseWhile(
+        () => !mergeabilityResolved,
+        () => octokit.pulls.get({
+            owner: GITHUB_OWNER,
+            repo: EXPENSIFY_CASH_REPO,
+            pull_number: pullRequestNumber,
+        })
+            .then(({data}) => {
+                if (!_.isNull(data.mergeable)) {
+                    mergeabilityResolved = true;
+                    isMergeable = data.mergeable;
+                }
+            })
```

Comment:
Should we log here that we saw null? Also should we add a sleep and/or max loop param? I feel like we could end up hammering the API on accident.

Reply:
Sure, which do you think is better? Sleep or max loop? Both?

Reply:
Maybe we should start out with a sane max default like: 10 second sleep, max 10 retries? What do you think?

Reply:
Just added a `_.throttle` for now, along with a log that we saw null

Reply:
I also added a `MAX_RETRIES` of 30 and throttle of 5 seconds. - 2 minutes 30 seconds total wait time before we give up.